app/scripts/loadout/dimLoadout.directive.js

```diff
@@ -3,9 +3,10 @@
 
   angular.module('dimApp').directive('dimLoadout', Loadout);
 
-  Loadout.$inject = ['dimLoadoutService'];
+  Loadout.$inject = ['dimLoadoutService','dimSettingsService'];
 
   function Loadout(dimLoadoutService) {
```

Comment:
oops, that was my bad.  I forgot that code was in place and didn't finish cleaning it up.

Fixed